sandbox/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:12.18.0-alpine3.10
+FROM node:16.14.2-alpine3.14
 
```

Comment:
In ERSSUP-61133, it was mentioned that npm - 8.5.0 would also be upgraded. I assume that npm - 8.5.0 is automatically upgraded as part node:16.14.2 upgrade. Is that so?

Reply:
Usually npm comes with Node.js

We can check the versions with

$ npm -v
$ node -v

Reply:
Yup they're both packaged together, node.js 16.14.2 is bundled with 8.5.0.